$OpenBSD$

Fix build, off_t is 64 bit, i.e. no need for nonexistent ftello64().

XXX CXXFLAGS are not propagated when building this, so -DIOAPI_NO_64
does not work.

Index: Telegram/ThirdParty/minizip/ioapi.c
--- Telegram/ThirdParty/minizip/ioapi.c.orig
+++ Telegram/ThirdParty/minizip/ioapi.c
@@ -14,7 +14,7 @@
         #define _CRT_SECURE_NO_WARNINGS
 #endif
 
-#if defined(__APPLE__) || defined(IOAPI_NO_64)
+#if defined(__APPLE__) || defined(IOAPI_NO_64) || defined(__OpenBSD__)
 // In darwin and perhaps other BSD variants off_t is a 64 bit value, hence no need for specific 64 bit functions
 #define FOPEN_FUNC(filename, mode) fopen(filename, mode)
 #define FTELLO_FUNC(stream) ftello(stream)
